classes/local/store/object_file_system.php

```diff
@@ -357,16 +357,20 @@ public function delete_empty_dirs($rootpath = '') {
      * @return void
      */
     public function readfile(\stored_file $file) {
-        $path = $this->get_remote_path_from_storedfile($file);
+        if ($this->is_configured()) {
```

Comment:
wondering if this would read better as

```
if ($this->is_configured()) {
    return parent::readfile($file);
}

...
```

we can repeat this pattern in a few places, ie supports_xsendfile should be the same (ie it can do a xsendfile for filedir if it is turned on)

Reply:
I think you meant
```
if (!$this->is_configured()) {
    return parent::readfile($file);
} else {
   //objectfs stuff
}
```
`if (!$this->is_configured()) {` instead of `if ($this->is_configured()) {`